src/components/views/rooms/MemberList.js

```diff
@@ -325,27 +348,82 @@ module.exports = React.createClass({
         });
     },
 
-    memberSort: function(userIdA, userIdB) {
-        var userA = this.memberDict[userIdA].user;
-        var userB = this.memberDict[userIdB].user;
+    memberString: function(member) {
+        if (!member) {
+            return "(null)";
+        }
+        else {
+            return "(" + member.name + ", " + member.powerLevel + ", " + member.user.lastActiveAgo + ", " + member.user.currentlyActive + ")";
+        }
+    },
 
-        var presenceMap = {
-            online: 3,
-            unavailable: 2,
-            offline: 1
-        };
+    memberSort: function(now) {
+        return (userIdA, userIdB) => {
+            // order by last active, with "active now" first.
```

Comment:
so, this returns > 0 if B should come before A

I realise this is the convention for sorting functions, but it would really save my brain cycles if those words were written down explicitly.